WordPress/Classes/ViewRelated/Stats/Insights/Insights Management/CustomizeInsightsCell.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+class CustomizeInsightsCell: UITableViewCell, NibLoadable {
+
+    // MARK: - Properties
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var contentLabel: UILabel!
+    @IBOutlet weak var dismissButton: UIButton!
+    @IBOutlet weak var tryButton: UIButton!
+    @IBOutlet weak var bottomSeparatorLine: UIView!
+
```

Comment:
You can remove the _weak_ here and make them private if you don't need to expose these properties.